RELEASE/scripts/autoscend/quests/level_05.ash

```diff
@@ -88,7 +101,12 @@ boolean L5_haremOutfit()
 }
 
 boolean L5_goblinKing()
-{
+{		
+	if (is_banished($phylum[goblin]) && get_property("screechCombats").to_int() > 0)
```

Comment:
there are only goblins in the harem / king location, so if they are banished they will all come back, so adventuring is fine